### PR TITLE
TipsForSlack: updated some language around `@here` guidelines.

### PR DESCRIPTION
**Pull request by cotarg**
I've drafted some clarifying language around Slack notifications && `@here`.

### CODE REVIEW

File: TipsForSlack.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Here’s an overview of the differences:
...
 
-For non-urgent announcements that you want to bring to the attention of everyone who is active in the slack channel, @here is best. It will alert those of us who are already online and in the slack, but it won’t interrupt folks who are not online.
+For non-urgent announcements that you want to bring to the attention of everyone who is active in the slack channel, @here is best. It will alert those of us who are already online and in the slack, but it won’t interrupt folks who are not online. Keep in mind that folks who are not online when you send the @here, but who log in later, will see the notification when they log in and become active! Also, please help fight alert fatigue by considering whether you need to notify multiple times for the same announcement.
```
Comment (lizrush) [inline TipsForSlack.md]: I like this line "Also, please help fight alert fatigue by considering whether you need to notify multiple times for the same announcement"--"alert fatigue" is a great way to put it. What if we just said that you should only do one alert per announcement? That way we could help clear up that it's fine to use @-here once, but not if you are cross posting to 5 channels all with @-here announcements?

Comment (lizrush) [inline TipsForSlack.md]: Can probably change this `Keep in mind that folks who are not online when you send the @here, but who log in later, will see the notification when they log in and become active`

to 
`Keep in mind that folks who are not online when you send the @here notification will see it when they log in and become active`

Comment (cotarg) [reply TipsForSlack.md]: 👍🏼

**Review by lizrush — APPROVED** [review]

**Comment by lizrush** [comment]
ty @cotarg 🙏

**Pull request merged**